fix: Fix loading-state container reference in loading-states extension

jozefrebjak opened the pull request:
## Description

Applied fix as in https://github.com/bigskysoftware/htmx/commit/695fd8aa14404afc695ae96949ce61965fc2b3ae for loading-states extension.

Corresponding issue: https://github.com/bigskysoftware/htmx/issues/2450

## Testing

I tested it locally on MacOS M1 Sonoma with Safari 17.4.1

## Checklist

* [x] I have read the contribution guidelines
* [x] I have targeted this PR against the correct branch (`master` for website changes, `dev` for
  source changes)
* [x] This is either a bugfix, a documentation update, or a new feature that has been explicitly
  approved via an issue
* [ ] I ran the test suite locally (`npm run test`) and verified that it succeeded

andryyy commented:
Let me do my part as a good user:

Can this be merged? Why hasn’t this been merged yet? Is the project dead?
Is there a fork? Why are the devs not responding?

Thanks for the fix. :)

jozefrebjak commented:
Dear @1cg ,

I would greatly appreciate it if you could review the current PR. Additionally, there are several other PRs specifically addressing issues related to Safari. 

Thank you in advance for your time.

Telroshan commented:
Hey, following up late on this ; since extensions have moved to [another repo](https://github.com/bigskysoftware/htmx-extensions) for v2, you'll want to either (ideally both if you feel like it!) retarget this PR to the `v1` branch, or make another PR on that extensions repo!
